docs/ralph/getting-started.md

```diff
@@ -128,6 +128,37 @@ Contract Baz() {
 }
 ```
 
+In Ralph, structs are value types, not reference types. When assigned to a new struct variable, it copies all fields to the new struct variable:
+
+```rust
+struct Foo { x: U256, mut y: U256 }
+Contract Bar() {
+  pub fn bar() -> () {
+    let foo0 = Foo { x: 0, y: 1 }
+    let mut foo1 = foo0
+    foo1.x = 1 // modifying foo1.x will not change foo0.x
+    assert!(foo0.x == 0, 0)
+    assert!(foo1.x == 1, 0)
+  }
+}
+```
+
+Ralph also supports struct destruction, and the syntax is similar to TypeScript:
+
+```rust
+struct Foo { x: U256, y: U256 }
+Contract Bar() {
+  pub fn bar() -> () {
+    let foo = Foo { x: 0, y: 1 }
+    let Foo { mut x, y } = foo
+    x = 1
+
+    // the variables `x` and `y` already exist, we create two new variables `x1` and `y1`
+    let Foo { x: x1, y: y1 } = foo
+  }
+}
+```
+
 ### Map (Rhone Upgrade Only)
 
 The map data structure is currently available exclusively in the [devnet release](https://github.com/alephium/alephium-stack/tree/master/rhone-devnet) of full node.
```

Comment:
We could update this to both testnet and devnet now

Reply:
Updated, but there is no `testnet` in `alephium-stack`.

Reply:
Will make one!